eq-author/src/components/buttons/Button/__snapshots__/index.test.js.snap

```diff
@@ -277,6 +277,215 @@ exports[`components/Button renders according to variant 4`] = `
 </button>
 `;
 
+exports[`components/Button renders according to variant 5`] = `
```

Comment:
Whats a greyed button? Doesn't seem to be used anywhere?

Reply:
was a new variant for the reply button - now removed until we implement reply